src/pages/EditMonitor.vue

```diff
@@ -265,18 +265,18 @@
                             <!-- SNMP Monitor Type -->
                             <div v-if="monitor.type === 'snmp'" class="my-3">
                                 <label for="snmp_community_string" class="form-label">{{ $t("Community String") }}</label>
-                                <input id="snmp_community_string" v-model="monitor.snmpCommunityString" type="text" class="form-control" required placeholder="public">
+                                <!-- TODO: Rename monitor.radiusPassword to monitor.password for general use -->
```

Comment:
Do you want me to address this in this PR?

Reply:
Lets do this in another PR.
This would need a migration and the PR is already somewhat big ^^

Reply:
Sounds good!